### DISCUSSION TITLE
Filter Custom on Page for all widgets

### DISCUSSION THREAD

**Discussion by AFSDARIF** [filamentphp/filament#6834] [Ideas]
I think this is so helpful, for make filter custom like date range picker on Page and all widget is filtered. Thanks before for making this Filament so cool.

**Comment by danharrin** [comment]
Looking to implement this in v3 as its now possible in Livewire v3

**Reply by AFSDARIF** [reply]
Okay thanks. I hope Filament v3 can run smoothly in the development process.
